internal/store/driver.go

```diff
@@ -910,6 +910,10 @@ func (d *Driver) getPolicyJSON(ingress *netv1.Ingress, modSet *ingressv1alpha1.N
 		return trafficPolicy.Spec.Policy, nil
 	}
 
+	if modSet == nil {
+		return policyJSON, nil
```

Comment:
I ran this last change through all the combinations and it's good to go: `ingress + module set`, `ingress + trafffic policy`, `ingress + module set + traffic policy` and just `ingress`. They behaved as expected in logs and synced to ngrok edges